tests/lib/rules/class-order.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview enforce ordering of classes
+ * @author Maciej Chmurski
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+var rule = require('../../../lib/rules/class-order')
+var RuleTester = require('eslint').RuleTester
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+var tester = new RuleTester({
+  parser: 'vue-eslint-parser',
+  parserOptions: { ecmaVersion: 2015 }
+})
+tester.run('class-order', rule, {
+
+  valid: [
+    {
+      filename: 'test.vue',
+      code: '<template><div></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div class="a b"></div></template>'
```

Comment:
Could you add testcases with multiple spaces?

```js
'<template><div class="a        b"></div></template>'
```

Reply:
done